cmd/contextShow.go

```diff
@@ -0,0 +1,34 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/netsage-project/grafana-dashboard-manager/config"
+	"github.com/sirupsen/logrus"
```

Comment:
Other files in this PR rename logrus to log. I suggest adding that here for consistency.

Reply:
Done